libs/designsystem/src/lib/scss/base/_lists.scss

```
@@ -0,0 +1,89 @@
+$bullet-font-size: 1.3em;
+$hollow-bullet-font-size: 0.8em;
+$list-item-line-height: 1.2em;
```

Remove unit from variable:
```suggestion
$list-item-line-height: 1.2;
```